README.md

```diff
@@ -1 +1,88 @@
-# angular-concepts-readme
+# Concepts
+
+## Overview
+
+There are some core concepts that every Angular developer will need to familiarise themselves with. This will teach you the core concepts that you will be using in every Angular application you make.
+
+## Objectives
+
+- Define MVC, MVVM
```

Comment:
spell out fully on first use - "Model-View-Controller (MVC) and Model-View-ViewModel (MVVM).